adserver/tasks.py

```diff
@@ -0,0 +1,52 @@
+"""Celery tasks for the ad server."""
+import logging
+
+from django.db.models import Count
+
+from .models import GeoImpression
+from .models import Offer
+from .utils import get_ad_day
+from config.celery_app import app
+
+log = logging.getLogger(__name__)  # noqa
+
+
+@app.task()
+def daily_update_geos():
```

Comment:
This should probably eventually take a `day` or `date_range` argument, but just wanted to keep it simple for now.

Reply:
It definitely should since without this we won't be able to run it on past data.